react-carousel-component/my-app/src/Indicators.jsx

```diff
@@ -0,0 +1,31 @@
+import { FaRegCircle, FaCircle } from 'react-icons/fa';
+
+type props = {
+  count: Number,
```

Comment:
You want to use the lowercase names for standard data types ("primitive types"): number, string, etc. The uppercase names are not primitive types and they are almost never used. The TS docs say:

> The type names String, Number, and Boolean (starting with capital letters) are legal, but refer to some special built-in types that will very rarely appear in your code. Always use string, number, or boolean for types.